views/peeps/index.erb

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Peeps</title>
+    <link rel="stylesheet" type="text/css" href="/style.css" />
+  </head>
+  <body>
+
+    <div class="forepage">
+      <h1> Chitter Board </h1>
+
+      <% if @user %>
+        <p>Signed in as <%= @user.name %></p>
+        <a href="http://localhost:9292/peeps/new">Make a peep</a>
```

Comment:
Maybe make these links relative? The app won't always be running on http://localhost:9292/, so "/peeps/new" would be better.